packages/strapi-generate-new/lib/before.js

```diff
@@ -284,7 +283,7 @@ module.exports = (scope, cb) => {
         }
 
         console.log();
-        console.log('⏳ Testing database connection...');
+        console.log(isQuick ? '✅ Connected to the database' : '⏳ Testing database connection...');
```

Comment:
Last time we discuss to add a message to say it can take a minute to download and test database connection.
I think we can add it in this PR - We have to define the exact message.

Reply:
Changes have to be done in the `./bin/start.js` to update `have a coffee` typo

Reply:
There is no test if you choose the quickstart option so the message seems correct, no?

Reply:
Yap but I talk about other databases.